lib/android-tools-info.ts

```diff
@@ -235,5 +318,24 @@ export class AndroidToolsInfo implements IAndroidToolsInfo {
 	private getMaxSupportedVersion(): number {
 		return this.parseAndroidSdkString(_.last(AndroidToolsInfo.SUPPORTED_TARGETS.sort()));
 	}
+
+	private validateAndroidHomeEnvVariable(androidHomeEnvVar: string): IFuture<boolean> {
```

Comment:
Usually validate methods return `true` when validation is successful and `false` otherwise.